app/controllers/kudos_controller.rb

```diff
@@ -71,6 +71,23 @@ def create
     end
   end
 
+  def update
```

Comment:
Yep! I think you can even change the line `kudo.update!(body: kudo_params[:body])` to something like `kudo.update_attributes!(kudo_params)`. The benefit is that you'd be able to use this endpoint later on for updating other things only by updating the strong parameters list. 

However as it is, it will let the user update `receiver_email` — why is that listed in the strong parameters?